hack/test/osctl-cluster-create.sh

```diff
@@ -27,7 +27,7 @@ run() {
 	 	k8s.gcr.io/hyperkube:${KUBERNETES_VERSION} -c "${1}"
 }
 
-${OSCTL} cluster create --name integration --image ${TALOS_IMG} --mtu 1440
+${OSCTL} cluster create --name integration --image ${TALOS_IMG} --mtu 1440 --cpus 4.0
```

Comment:
Sorry. My refactor moved this to the `basic-integration.sh` script.

Reply:
wow, git was able to rebase it automatically